webui/react/src/components/ProjectCard.tsx

```diff
@@ -49,9 +49,9 @@ const ProjectCard: React.FC<Props> = ({
       onDropdown={onClick}>
       <div className={classnames.join(' ')}>
         <div className={css.headerContainer}>
-          <Typography.Title className={css.name} ellipsis={{ rows: 3, tooltip: true }} level={5}>
+          <Title size={TypographySize.XS} truncate={{ rows: 3, tooltip: true }}>
```

Comment:
I didn't notice this before, but I think 3 rows of truncation increases the card height in a way that looks awkward. maybe change this to 1 row?

Reply:
✅

Reply:
I think `.headerContainer` needs a width applied to it. not sure why that didn't matter with 3 rows but it's not truncating.